test/dev_bundles/dev.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>StealJS development bundles</title>
+</head>
+<body>
+	<script
+		dev-bundle
+		main="@empty"
+		src="../../steal.js"
+		dev-bundle-dest="test/dev_bundles/"
```

Comment:
I was thinking that `dev-bundle` and `dev-bundle-dest` should just be one option.

The way I would implement this is to have code [just like this](https://github.com/stealjs/steal/blob/master/src/config.js#L35):

```js
System.devBundle = "@empty";
```

Then setup a setter [on this object](https://github.com/stealjs/steal/blob/master/src/config.js#L180)

It would make it so that `dev-bundle` attribute becomes:

```js
System.devBundle = "dev-bundle";
```

And if the user types `dev-bundle="test/dev_bundles/dev-bundle"`

Then it becomes:

```js
System.devBundle = "test/dev_bundles/dev-bundle";
```

This way in startup.js you can just directly do `System.import(System.devBundle)` and not have to worry about checking if it exists and doing any sort of pathing stuff. It will be `@empty` by default, otherwise it goes through the normal config pipeline to set its moduleName.